manifests/m/Microsoft/PowerToys/0.94.1/Microsoft.PowerToys.installer.yaml

```diff
@@ -0,0 +1,59 @@
+# Created with komac v2.13.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.10.0.schema.json
+
+PackageIdentifier: Microsoft.PowerToys
+PackageVersion: 0.94.1
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.17134.0
+InstallerType: burn
+InstallerSwitches:
+  Silent: /quiet /norestart
+  SilentWithProgress: /quiet /norestart
+UpgradeBehavior: install
+ReleaseDate: 2025-09-17
+Installers:
+- Architecture: x64
+  Scope: user
+  InstallerUrl: https://github.com/microsoft/PowerToys/releases/download/v0.94.1/PowerToysUserSetup-0.94.1-x64.exe
+  InstallerSha256: E69C4E506FB1081F52A45E46FDBAC1EC198F4D83875DE025B1A6856F0A9436EF
+  AppsAndFeaturesEntries:
+  - DisplayName: PowerToys (Preview) x64
+    ProductCode: '{C4525AE6-6796-4357-9DAF-F9191FF3B1A6}'
+    UpgradeCode: '{6341382D-C0A9-4238-9188-BE9607E3FAB2}'
+    InstallerType: burn
+- Architecture: x64
+  Scope: machine
+  InstallerUrl: https://github.com/microsoft/PowerToys/releases/download/v0.94.1/PowerToysSetup-0.94.1-x64.exe
+  InstallerSha256: E83FF71F951C32218F4F271CD3C01F8B354FDDCA16CAE74D36025D2D41296C8A
+  AppsAndFeaturesEntries:
+  - DisplayName: PowerToys (Preview) x64
+    ProductCode: '{CB07F47A-2358-4FB7-9A88-5588DEA7D612}'
+    UpgradeCode: '{6341382D-C0A9-4238-9188-BE9607E3FAB2}'
+    InstallerType: burn

```

Comment:
We don't need the extra ARP entries & `InstallerType: burn` is already defined at the root so no need to redefine it at installer level. Also, we have another PR from the official PowerToys team GitHub Action workflow so I'll close this in favor of that PR:
- https://github.com/microsoft/winget-pkgs/pull/294593